app/src/main/java/com/github/onlynotesswent/model/file/FileRepositoryFirebaseStorage.kt

```diff
@@ -0,0 +1,143 @@
+package com.github.onlynotesswent.model.file
+
+import android.net.Uri
+import android.util.Log
+import com.github.onlynotesswent.model.note.Note.Type
+import com.google.firebase.storage.FirebaseStorage
+import com.google.firebase.storage.StorageReference
+import java.io.File
+
+/** Maximum file size in bytes for downloading directly to memory: 50 MB */
+const val MAX_FILE_SIZE: Long = 50 * 1024 * 1024
```

Comment:
This can be a private const

Reply:
I left it public for testing (test if the file surpasses max file size), and I feel it can be useful to detect from the outside (to show in the UI what the max file size is for example)